ArduPlane/quadplane.h

```diff
@@ -196,6 +197,13 @@ class QuadPlane
     // air mode state: OFF, ON, ASSISTED_FLIGHT_ONLY
     AirMode air_mode;
 
+    // Command model parameter class
+    // Default max rate, default expo, default time constant
+    AC_CommandModel command_model_pilot{100.0, 0.25, 0.25};
```

Comment:
@IamPete1 I left the default for copter to zero for the time constant.  That way it does not change users set ups.  It functions just as it did before the addition of the time constant.  It  is up to you of course for quadplane.  Do you find 0.25 seconds to work well?  It seems a little slow by heli standards but I presume quadplanes don't have as much control power in the yaw axis

Reply:
It seemed to work OK, only tested in the simulator tho. Quadplanes tend to be larger than the copters/helis and have much worse yaw authority. Hopefully we will get some feedback from users once its merged.